entrypoint.sh

```diff
@@ -2,6 +2,7 @@
 output_file=${OUTPUT_FILE:-lsif-output.json}
 skip=${SKIP:-False}
 
+git config --global --add safe.directory ${GITHUB_WORKSPACE}
```

Comment:
Could we potentially just add this to the step that runs before the parsers?

Reply:
I tried that first, but it didn't appear to help -- either because the path of the git repo is different between the GH Actions workflow (`/home/runner/work/python/python`) and the parser container (`/github/workspace`), or because the global git config is saved somewhere under `$HOME` and that doesn't get used in the parser container.  Either way, this seems to work, and perhaps we can get rid of it later on.